apps/data-scanner/lib/api/rest-api/router.mjs

```diff
@@ -1,9 +1,12 @@
-import { Router } from 'express';
-import { sessions } from './routes/index.mjs';
+import {Router} from 'express';
+import {scanner} from "../app.mjs";
 
 const router = Router();
 
 router.get('/', (req, res) => res.send('Hello'));
-router.get('/sessions_update', sessions.update);
+router.get('/sessions_sync', async (req, res) => {
```

Comment:
implement the same logic with makeRequestHandler
add new UseCase for sync and use await appCommand.invoke inside this usecase